src/install/uninstall

```diff
@@ -0,0 +1,10 @@
+read -e -p "The script will uninstall sourcerer app. Proceed? [Y/n] " yesno < /dev/tty
```

Comment:
I confirm that I didn't miss this comment and I think it is reasonable. Not yet sure, why I am doing this :)